tools/tool_settings.h

```diff
@@ -819,6 +820,11 @@ GetVulkanReplayOptions(const gfxrecon::util::ArgumentParser&           arg_parse
         replay_options.enable_use_captured_swapchain_indices = true;
     }
 
+    if (arg_parser.IsOptionSet(kEnableOffscreen))
+    {
+        replay_options.enable_offscreen = true;
```

Comment:
Good question. We should combine `offscreen` and `captured_swapchain_indices` into one option. Like `--swpachain virtual`, `--swpachain captured_indices`   and `--swpachain offscreen`.   `--swpachain virtual` is default. What do you think?

Reply:
That is OK with me, but we may need to keep `--use-captured-swapchain-indices` to maintain backward compatibility. @bradgrantham-lunarg What do you think?

Reply:
I think we need to keep `--use-captured-swapchain-indices`.

> Existing environment variables and properties for capture as well as command line arguments to replay would continue to work as implemented or be disabled with a non-fatal message so that scripts would not unexpectedly break.

I'm okay with `--use-captured-swapchain-indices` becoming a synonym for `--swapchain offscreen`.

Reply:
If this change is made, would it be accurate enough to name the `captured-indices` option just `captured`, so it would be: `--swapchain captured`, `--swapchain virtual` or `--swapchain offscreen`?

Reply:
Sure. It would. If users set `--swapchain`, it will ignore `--use-captured-swapchain-indices`.

Reply:
Added!